src/app/core/data/href-only-data.service.ts

```diff
@@ -0,0 +1,92 @@
+import { DataService } from './data.service';
+import { RequestService } from './request.service';
+import { RemoteDataBuildService } from '../cache/builders/remote-data-build.service';
+import { Store } from '@ngrx/store';
+import { CoreState } from '../core.reducers';
+import { ObjectCacheService } from '../cache/object-cache.service';
+import { HALEndpointService } from '../shared/hal-endpoint.service';
+import { NotificationsService } from '../../shared/notifications/notifications.service';
+import { HttpClient } from '@angular/common/http';
+import { DefaultChangeAnalyzer } from './default-change-analyzer.service';
+import { Injectable } from '@angular/core';
+import { VOCABULARY_ENTRY } from '../submission/vocabularies/models/vocabularies.resource-type';
+import { FindListOptions } from './request.models';
+import { FollowLinkConfig } from '../../shared/utils/follow-link-config.model';
+import { dataService } from '../cache/builders/build-decorators';
+import { RemoteData } from './remote-data';
+import { Observable } from 'rxjs/internal/Observable';
+import { PaginatedList } from './paginated-list.model';
+import { ITEM_TYPE } from '../shared/item-relationships/item-type.resource-type';
+import { LICENSE } from '../shared/license.resource-type';
+import { CacheableObject } from '../cache/object-cache.reducer';
+
+/* tslint:disable:max-classes-per-file */
+class DataServiceImpl extends DataService<any> {
+  // linkPath isn't used if we're only searching by href.
+  protected linkPath = undefined;
+
+  constructor(
+    protected requestService: RequestService,
+    protected rdbService: RemoteDataBuildService,
+    protected store: Store<CoreState>,
+    protected objectCache: ObjectCacheService,
+    protected halService: HALEndpointService,
+    protected notificationsService: NotificationsService,
+    protected http: HttpClient,
+    protected comparator: DefaultChangeAnalyzer<any>) {
+    super();
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+@dataService(VOCABULARY_ENTRY)
+@dataService(ITEM_TYPE)
+@dataService(LICENSE)
+export class HrefOnlyDataService {
```

Comment:
Please add some TypeDocs to describe this new service.

Also, this new service doesn't appear to have any specs?

Reply:
Added docs and specs